src/wrappers/ReactWrapper.tsx

```diff
@@ -117,6 +125,9 @@ export const ReactWrapper = {
       (this as any).reactComponentRef.setState({ children: null });
     }
   },
+  reactRef() : any {
+    return (this as any).reactComponentRef;
```

Comment:
I guess we break any autocompletion then. Would have to figure out some generic typing here.